.travis.yml

```diff
@@ -24,6 +24,6 @@ before_script:
 script:
   - go vet -mod=vendor ./...
   - go test -mod=vendor -v -race -covermode=atomic -coverprofile=coverage.out -tags=integration ./...
-  - goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
-  - ct-woodpecker -config ./test/config/ct-woodpecker.localdev.json &
+  - goveralls -coverprofile=coverage.out -service=travis-ci
+  - ct-woodpecker -config ./test/config/config.json &
```

Comment:
This diff to the `-config` argument doesn't seem right. https://github.com/letsencrypt/ct-woodpecker/pull/76 renames that file to `ct-woodpecker.localdev.json`.

It causing this branch to fail CI:
> 2018/12/19 20:21:12 Unable to load ct-woodpecker config: open ./test/config/config.json: no such file or directory

Reply:
Ahhh! That makes sense. I would probably just fix the diff as an add-on commit in this branch with a message explaining its fixing a merge error.